apps/summerfi-api/lib/sdk-router-function/e2e/refinance-e2e.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+import { createTRPCClient, httpBatchLink } from '@trpc/client'
+import { Protocols, type Position, type PositionId, type Protocol } from '../src/sdk-mocks'
```

Comment:
With the separated tsconfig and tsconfig.build version, you can now import with `~sdk/sdk-mocks`

Reply:
thanks done